src/sema/expression/function_call.rs

```diff
@@ -547,20 +547,19 @@ pub(super) fn function_call_named_args(
     Err(())
 }
 
-/// Resolve a method call with positional arguments
-pub(super) fn method_call_pos_args(
+/// Check if the function is a method of a variable
```

Comment:
Please explain what all the different variants of the return type mean

Reply:
Do you mean `Result<Option<Expression>, ()>`?